### ISSUE TITLE
gremlin 3.5 (dependabot proposal) causes breakage

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#5297]
In recent dependabot PRs, and update to gremlin 3.5.0 has been proposed.

However in testing for #5291 a variety of problems were noted
 - open lineage graph connector needs to use org.apache.commons:commons-configuration2 instead of commons-configuration:commons-configuration (minor import change)
  - Some FVTs using the graph repository fail. This includes a) not initializing (classpath?), issues with searches (bug, or bug fix?)
  
  Opening issue to relook at this and capture exact data, Sticking with gremlin 3.4.x until resolved (and until other dependency updates are merged so as to get a clean comparison)

**Comment by planetf1** [comment]
See #5275

**Comment by planetf1** [comment]
Example failure:
```
DataManagerOMASCreateDatabaseIT > testCreateDB(String) > org.odpi.openmetadata.accessservices.datamanager.fvt.execution.DataManagerOMASCreateDatabaseIT.testCreateDB(String)[2] STANDARD_OUT
    ==========================================
    CreateDatabaseTest FVT failed
    Number of Failed tests: 1
    Captured Exceptions: 1
     ==> org.odpi.openmetadata.fvt.utilities.exceptions.FVTUnexpectedCondition with message createDatabaseTable
    FVTUnexpectedCondition{testCaseName='CreateDatabaseTest', errorDescription='createDatabaseTable', caughtException=PropertyServerException{reportedHTTPCode=500, reportingClassName='org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler', reportingActionDescription='getDatabaseTablesByName', reportedErrorMessage='OMAG-REPOSITORY-HANDLER-500-001 An unexpected error org.odpi.openmetadata.repositoryservices.ffdc.exception.RepositoryErrorException was returned to getEntitiesByName by the metadata server during getDatabaseTablesByName request for open metadata access service Data Manager OMAS on server servergraph; message was OMRS-REPOSITORY-400-055 An unexpected java.lang.ClassCastException exception was received from a repository connector during the findEntitiesByProperty operation which had message: class org.apache.tinkerpop.gremlin.structure.util.empty.EmptyGraph cannot be cast to class org.janusgraph.graphdb.database.StandardJanusGraph (org.apache.tinkerpop.gremlin.structure.util.empty.EmptyGraph and org.janusgraph.graphdb.database.StandardJanusGraph are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @1b604f19)', reportedErrorMessageId='OMAG-REPOSITORY-HANDLER-500-001', reportedErrorMessageParameters=[OMRS-REPOSITORY-400-055 An unexpected java.lang.ClassCastException exception was received from a repository connector during the findEntitiesByProperty operation which had message: class org.apache.tinkerpop.gremlin.structure.util.empty.EmptyGraph cannot be cast to class org.janusgraph.graphdb.database.StandardJanusGraph (org.apache.tinkerpop.gremlin.structure.util.empty.EmptyGraph and org.janusgraph.graphdb.database.StandardJanusGraph are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @1b604f19), getDatabaseTablesByName, Data Manager OMAS, servergraph, org.odpi.openmetadata.repositoryservices.ffdc.exception.RepositoryErrorException, getEntitiesByName], reportedSystemAction='The system is unable to process the request because of an internal error.', reportedUserAction='Verify the sanity of the server.  This is probably a logic error.  If you can not work out what happened, ask the Egeria community for help.', reportedCaughtException=null, reportedCaughtExceptionClassName='null', relatedProperties=null}}

```

As per https://github.com/JanusGraph/janusgraph/pull/2619, this will be addressed in JanusGraph 0.6. As such will close this issue and ignore the dependabot report for now
